app/konnect/runtime-manager/composite-runtime-groups/index.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Composite Runtime Groups
+content_type: explanation
+badge: enterprise
+---
+
+A composite runtime group is a read-only group that combines configuration from
+its members, which are standard runtime groups. All of the standard runtime groups within a 
+composite runtime group share the same cluster of runtime instances. 
+
+![Standard runtime group](/assets/images/docs/konnect/konnect-standard-rg.svg)
+> _**Figure 1:** In a standard runtime group setup, each team configures and manages their own runtime instances._
+_For example, Team Blue configures Runtime Group Blue, which then uses a set of runtime instances that only run Blue configuration._
+
+
+![Composite runtime group](/assets/images/docs/konnect/konnect-composite-rg.svg)
+> _**Figure 2:** In a composite runtime group setup, each team still configures their own runtime groups, but the runtime instances are shared._
+_For example, Team Blue configures Runtime Group Blue, which is then combined with the configuration from Team Green and Team Yellow._
+_The runtime instances in the cluster use the combined configuration._
```

Comment:
suggestion: We should explain what the green runtime group is doing in this diagram since Team Green is not present. The story we are telling here is of a federated platform administrator model where a central platform team administers the composite runtime group while individual service teams administer their own member runtime group. The green runtime group represents a way for a central platform team to configure global plugins on all the configuration running in the composite runtime group. (Unsure if we want to go into this detail in a caption though, your choice)

Reply:
I was already thinking that I might turn this into a regular paragraph description instead of just a caption. Makes it flow better. I'll add in this info.